PhysicsTools/JetExamples/test/BuildFile.xml

```diff
@@ -2,23 +2,20 @@
 <library file="printJetFlavourInfo.cc">
   <flags EDM_PLUGIN="1"/>
   <use name="FWCore/Framework"/>
-  <use name="SimDataFormats/JetMatching"/>
```

Comment:
Instead of removing all these, shouldn't they get replaced by `DataFormats/JetMatching`?